sysdig/resource_sysdig_secure_policy.go

```diff
@@ -258,7 +259,9 @@ func resourceSysdigPolicyRead(ctx context.Context, d *schema.ResourceData, meta
 
 	if err != nil {
 		d.SetId("")
-		return diag.FromErr(err)
+		if !strings.Contains(err.Error(), policyNotFound) {
```

Comment:
Is it possible to use the status code 404 rather than the error message? The only concern is that someone might not know that the error message is being used by the terraform provider and unintentionally breaks it

Reply:
The way it is setup, the status code is not available at this level. And I am worried about making changes beyond this area due to lack of understanding about how it would affect other components.